config/provisioners/in-memory-channel/in-memory-channel.yaml

```diff
@@ -43,6 +43,12 @@ rules:
       - list
       - watch
       - update
+  - apiGroups:
```

Comment:
Isn't this a strict subset of the permissions granted on lines 36-45?

```
   - apiGroups:
       - eventing.knative.dev # Line 37
     resources:
       - channels # Line 39
     verbs:
      - update # Line 45
```

Is there something special about finalizers that means it is not the in the normal `crd` resource, but needs to be in `crd/finalizers`?

Reply:
Unfortunately I cannot find references to understand how it should behave, but this issue occurs on OpenShift 3.11.0 and I suspect it can affect also some versions of Kubernetes, although I didn't test on Kube.